core/src/navigation/LeftNav.html

```diff
@@ -545,6 +551,22 @@
       : NavigationHelpers.prepareForTests(key || metaInfo.label);
   }
 
+  function resolveTooltipText(node, translation) {
+    if (node.hasOwnProperty('tooltipText')) {
+      if (node.tooltipText === undefined || node.tooltipText === null) {
+        return defaultsTooltipTextConfig;
+      } else if (node.tooltipText.length) {
+        return node.tooltipText;
+      }
+    }
+
+    if (node.tooltipText === false || node.tooltipText === '') {
+      return '';
+    }
+
+    return translation;
+  }
+
```

Comment:
This looks great already 👍 . Thanks for applying suggestions.
- I can see the methods became larger then I expected. Do you think it would make sense to move them to **core/src/utilities/helpers/navigation-helpers.js**. Got a similar request on my [PR](https://github.com/SAP/luigi/pull/2267#discussion_r706195896)  from @stanleychh , sounds like a similiar use case here.

Reply:
@ndricimrr Hmm good point. Actual we use it twice but maybe we will implement it in another place so we can move it to navigation-helpers.js.